toolchain/check/context.h

```diff
@@ -179,6 +165,14 @@ class Context {
     return import_ref_ids_;
   }
 
+  // Pre-computed parts of a binding pattern.
+  struct BindingPatternInfo {
+    // The corresponding AnyBindName inst.
+    SemIR::InstId bind_name_id;
+    // The region of insts that computes the type of the binding.
+    SemIR::ExprRegionId type_expr_region_id;
+  };
+
```

Comment:
Optional: I prefer to omit the vertical space between the type and the function, in an effort to help cue the reader that the comment applies to both of them.

Reply:
Done